wagtail/images/templates/wagtailimages/images/results.html

```diff
@@ -44,11 +44,14 @@ <h2 role="alert">{% blocktrans %}Sorry, no images match "<em>{{ query_string }}<
 
         {% search_other %}
     {% else %}
-        {% url 'wagtailimages:add_multiple' as wagtailimages_add_image_url %}
-        {% if current_collection %}
-            <p>{% blocktrans %}You haven't uploaded any images in this collection. Why not <a href="{{ wagtailimages_add_image_url }}">upload one now</a>?{% endblocktrans %}</p>
-        {% else %}
-            <p>{% blocktrans %}You haven't uploaded any images. Why not <a href="{{ wagtailimages_add_image_url }}">upload one now</a>?{% endblocktrans %}</p>
-        {% endif %}
+        {% url 'wagtailimages:add_multiple' as wagtailimages_add_image_url_base %}
+        {% querystring as querystring %}
+        {% with wagtailimages_add_image_url=wagtailimages_add_image_url_base|add:querystring %}
+          {% if current_collection %}
+              <p>{% blocktrans %}You haven't uploaded any images in this collection. Why not <a href="{{ wagtailimages_add_image_url }}">upload one now</a>?{% endblocktrans %}</p>
```

Comment:
@ababic querystring is added in line 48:

```
{% with wagtailimages_add_image_url=wagtailimages_add_image_url_base|add:querystring %}
```

I choose to add `querystring` in `with` tag rather than changing variable name as any change inside `blocktrans` tag, would require recreating translation catalogs. What do you think?

Reply:
@bmihelac Looking at this again, I see you did the right thing... I'd just misread this :)